src/Components/test/testassets/TestServer/PrerenderedStartup.cs

```diff
@@ -23,7 +23,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
             services.AddServerSideBlazor();
             services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme).AddCookie();
-            services.AddSingleton<LazyAssemblyLoader>();
+            services.AddScoped<LazyAssemblyLoader>();
```

Comment:
@captainsafia I had trouble running the TestServer app without changing this. `AddServerSideBlazor` adds a scoped `IJSRuntime` instance. Our DI system will complain if a singleton service (in this case `LazyAssemblyLoader`) references a scoped service (`IJSRuntime`). Do you think there's a regression as a result of my changes?